src/core/IO/Socket/INET.pm

```diff
@@ -89,6 +90,7 @@ my class IO::Socket::INET does IO::Socket {
                $family == PIO::PF_INET
             || $family == PIO::PF_INET6
         } = PIO::PF_INET,
+        Str :$source-address is copy = "".Str,
```

Comment:
I get an error on the nqp::connect call (can't unbox the Str to str) if I don't put .Str